### PR TITLE
Http_File consideration for warning in File class

### PR DESCRIPTION
**Pull request by Canta**
Please refer to #945 for further details.

### CODE REVIEW

File: packager/file/file.cc [hunk 1]
```diff
@@ -287,7 +287,9 @@ bool File::WriteFileAtomically(const char* file_name,
 
   // Skip the warning message for memory files, which is meant for testing
   // anyway..
-  if (strncmp(file_name, kMemoryFilePrefix, strlen(kMemoryFilePrefix)) != 0) {
+  // Also check for http files, as they can't do atomic writes.
+  if (strncmp(file_name, kMemoryFilePrefix, strlen(kMemoryFilePrefix)) != 0
+      && strncmp(file_name, kHttpFilePrefix, strlen(kHttpFilePrefix)) != 0) {
```
Comment (kqyang) [inline packager/file/file.cc:292]: Needs to check `kHttpsFilePrefix` too.

Comment (Canta) [reply packager/file/file.cc]: Oh, I didn't saw that var in `file.h`, so I missed it. 

Please take a look at the last commit. It's a quick commit, as I can't build shaka in my current device (I usually do test builds before commits), but should work anyways.

**Comment by Canta** [comment]
IDK what has happened with the CI job, doesn't seem to be code related:

https://ci.appveyor.com/project/shaka/shaka-packager/build/job/l6g18yj3w5b0l07f

```
Uploading artifacts...
[1/3] deploy\mpd_generator-win.exe (15,540,224 bytes)...1%
Error uploading artifact the storage: Unable to read data from the transport connection: The connection was closed.
```

Would you please take a look at it @kqyang ?
Perhaps it can be re-triggered?

**Comment by kqyang** [comment]
@Canta Looks like a buildbot issue. I have triggered the build again.

**Pull request merged**